tests/unit/cluster/cluster-multiple-meets.tcl

```diff
@@ -0,0 +1,83 @@
+# make sure the test infra won't use SELECT
+set old_singledb $::singledb
```

Comment:
This is not strictly needed, right? I saw the servers are started with `cluster-enabled`

Reply:
This is needed (for now at least) since normally we select a random db. We could key it off of the clustering tests.